<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PayModesWidget</class>
 <widget class="QWidget" name="PayModesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>525</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="top_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>32567</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <layout class="QHBoxLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="topMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <property name="bottomMargin">
       <number>11</number>
      </property>
      <item>
       <widget class="QLabel" name="caption_label">
        <property name="font">
         <font>
          <family>Nimbus Sans l [Urw]</family>
          <pointsize>18</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Payment Mode</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="caption_spacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>231</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="middle_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="total_receivable_frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32567</width>
          <height>60</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>11</number>
         </property>
         <property name="topMargin">
          <number>11</number>
         </property>
         <property name="rightMargin">
          <number>11</number>
         </property>
         <property name="bottomMargin">
          <number>11</number>
         </property>
         <item>
          <widget class="QLabel" name="total_receivable_text_label">
           <property name="font">
            <font>
             <family>Nimbus Sans l [Urw]</family>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Total Receivable:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="total_receivable_spacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>181</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="total_receivable_value_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32767</width>
             <height>32767</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Nimbus Sans l [Urw]</family>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0  â‚¬</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="paymode_main_layout">
        <item>
         <layout class="QVBoxLayout" name="paymode_sub_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="line1_layout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="cash_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../icons/payment/1000xaf.jpeg</normaloff>../../icons/payment/1000xaf.jpeg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="proteccion_civil_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../icons/payment/orange_money.svg</normaloff>../../icons/payment/orange_money.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="fourb_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../icons/payment/mtn_mobile_money.png</normaloff>../../icons/payment/mtn_mobile_money.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="line2_layout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="restaurant_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../icons/payment/airtel.svg</normaloff>../../icons/payment/airtel.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="mastercard_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../icons/payment/master_card.svg</normaloff>../../icons/payment/master_card.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="visa_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../icons/payment/visa.svg</normaloff>../../icons/payment/visa.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="line3_layout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="gourmet_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../icons/payment/google_pay.svg</normaloff>../../icons/payment/google_pay.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="amex_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../icons/payment/american_express.svg</normaloff>../../icons/payment/american_express.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="sodexho_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../icons/payment/paypal.svg</normaloff>../../icons/payment/paypal.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="cancel_line">
     <property name="frameShape">
      <enum>QFrame::Shape::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="botom_layout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="cancel_spacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>260</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
